doc/api/index.json

```diff
@@ -1,13 +1,68 @@
 {
   "resources": [
     {
-      "name": "Profiles",
+      "name": "v1 Bookmarks",
```

Comment:
Since you're using json schema, is the doc folder generated?

Reply:
Yeah, and there is a required `api.md` readme file in it too.

Reply:
Okay so maybe you shouldn't commit the files in the doc repo

Reply:
Apitome uses them to actually render the production docs, so they need to be around until I get a docker build setup that has the ability to regenerate the docs

Reply:
Its a little unfortunate